### ISSUE TITLE
Accessibility: Name property of "Print button" in Print Preview dialog must not include the same text as its LocalizedControlType/ControlType properties

### ISSUE THREAD

**Issue by M-Lipin** [dotnet/winforms#1353]
Steps to reproduce:
1. Create .NET Core WinForms application and add PrintPreviewDialog control to it.
2. Build and run it, open Print preview dialog.
3. Inspect the accessibility names of the dialog.

**Actual:**
The Name property of "Print button" in Print Preview dialog include the same text/element as its LocalizedControlType/ControlType properties and announced by Narrator as "Print button button"

![image](https://user-images.githubusercontent.com/23213980/61125733-45e13880-a4b3-11e9-96bb-0bf0720998b5.png)

Violation: Section 508 502.3.1

**Expected:** 
The Name property of "Print button" in Print Preview dialog must not include the same text as its LocalizedControlType/ControlType properties.
